app/models.py

```diff
@@ -1144,24 +1136,24 @@ def applications_closed_at(cls):
                 '2 weeks 23:59:59', INTERVAL))
 
     @hybrid_property
-    def clarification_questions_closed_at(self):
-        if self.published_at is None:
+    def clarification_questions_closed_at(self_or_cls):
```

Comment:
Not that experienced with the difference between property and hybrid properties but is another option here to knock it back to being a property? Therefore it won't even be possible to try and use it as a SQL expression?